website/docs/core-library/usage.md

```diff
@@ -7,7 +7,7 @@ slug: /core-library/usage
 
 ## API Versioning
 
-The public API of the H3 Core Library is defined in the file `h3api.h`. The functions defined in h3api.h adhere to [Semantic Versioning](http://semver.org/).
+The public API of the H3 Core Library is defined in the file [`h3api.h`](https://github.com/uber/h3/blob/master/src/h3lib/include/h3api.h.in). The functions defined in `h3api.h` adhere to [Semantic Versioning](http://semver.org/).
```

Comment:
Since we're linking to the `.in` file that we preprocess (not using the C preprocessor) maybe we need to explain that a bit?